examples/cpp/aot_inductor/llama2/src/llama2.so/llama2.hh

```diff
@@ -0,0 +1,72 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <ctype.h>
+#include <time.h>
+#include <math.h>
+#include <string.h>
+#include <fcntl.h>
+#include <unistd.h>
+#include <sys/mman.h>
+#include <torch/csrc/inductor/aoti_runner/model_container_runner_cpu.h>
```

Comment:
it seems that pt2.2 changed this file path.

Reply:
That is the current path. Nightlies are needed for this. README shows how to install nightlies with build.sh.
https://github.com/pytorch/pytorch/blob/main/torch/csrc/inductor/aoti_runner/model_container_runner_cpu.h

Reply:
In the build.sh, which version of libtorch should be installed in this release? nightly or PT2.2 ?

Reply:
For AOTInductor example the nightlies as described in the README.md https://github.com/pytorch/serve/blob/37608a2e8549071bbd457c96ca91f5c51375d802/examples/cpp/aot_inductor/llama2/README.md#setup